humilis_elasticache/resources.yaml.j2

```diff
@@ -1,6 +1,23 @@
 ---
 resources:
     CacheCluster:
+    {% if engine == "redis" and num_cache_nodes > 1 %}
+        Type: "AWS::ElastiCache::ReplicationGroup"
```

Comment:
To create a several node redis elasticache, the API to use  is not `CacheCluster` but `ReplicationGroup`.